### DISCUSSION TITLE
[Feature] Web UX: asset viewer: info pane: clicking location name should directly open external map

### DISCUSSION THREAD

**Discussion by pyorot** [immich-app/immich#10530] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

![image](https://github.com/immich-app/immich/assets/28405293/f998f9b6-0392-425d-9381-4914938dcc52)

The current behaviour is clicking the map pictogram, place name and pencil icon all load "change location", and clicking on the embedded map's marker shows a tooltip with "Open in <map provider>", which opens an external map.

I think it would be faster and more intuitive for the user if clicking the place name opened an external map. The pictogram could then not be a link, and the pencil and embedded map would work as before.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
You can now jump to the main map view with this button.

![image](https://github.com/user-attachments/assets/55a189e2-0d35-4da5-b340-0291e0378d21)
